### DISCUSSION TITLE
I just wanted to make my taskbar more transparent - i dont want any of those other changes!

### DISCUSSION THREAD

**Discussion by ThoMax011** [Open-Shell/Open-Shell-Menu#937] [Q&A]
I wasnt quite happy with the default windows 10 taskbar so I looked around to find a way to make it more transparent. First I tried the "UseOLEDTransparency"-method in the regestry-editor but that didnt work at all which confused me since it had worked on my old pc.
Then I found out about this tool and was amazed by all the adjustments that CAN be made with it. I emphasise "CAN" because the only thing I wanted to change is the damn taskbar. As I said Open Shell is a nice tool to play around with but I dont need all that other stuff.

Someone pls tell me how I can disable/revert all the changes that Open Shell made besides the taskbar?

I know I probably didnt need to ask this stupid question if I had just read more and paid more attention but I thought Id ask you guys directly since you know the most about it.

...transparent taskbar ...thats all I want :(

**Comment by e-t-l** [comment]
Open-Shell is probably overkill for your needs, then. I recommend uninstalling it and trying something simpler like Rainmeter, using the TranslucentTaskbar skin.

**Reply by ThoMax011** [reply]
Ok. Ive accepted my taskbar as it is and uninstalled open shell already. Thanks for the advice tho :)
